build-system/tasks/extension-helpers.js

```diff
@@ -178,6 +178,9 @@ function getExtensionsToBuild(preBuild = false) {
   if (extensionsToBuild) {
     return extensionsToBuild;
   }
+  if (argv.core_runtime_only) {
+    return (extensionsToBuild = []);
+  }
```

Comment:
@jridgewell This change breaks the ability to run `gulp dist --core_runtime_only --extensions=amp-foo` (added in #31091), which lets you compile `v0.js` and select extension(s), but skip all the other non-`v0.js` entry points.

Reply:
I'll send an update. I was tired of `--core_runtime_only` building the default extensions.